CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -250,40 +253,48 @@ function Panel3D.GetGUI(panel)
 end
 
 local zeroVector = Vector3.new(0, 0, 0)
-local baseHorizontal = CFrame.new()
-local basePosition = Vector3.new()
+local headHorizontal = CFrame.new()
```

Comment:
Perhaps instead of `horizontal` `XZPlane` is easier to understand? This is really up to you though.